curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65364566e84e378837fbaf2a.md

```diff
@@ -7,9 +7,11 @@ dashedName: step-28
 
 # --description--
 
-You need to hook up the `playSong` function to a `click` event listener so the songs will start playing when the play button is clicked.
+# --description--
```

Comment:
This looks like the same change from your other PR.
So you can remove it from this one